File: 0x12-javascript-warm_up/5-to_integer.js
#!/usr/bin/node
// prints 'My number:' is the first arguemnt can be converted into an int
let i = parseInt(process.argv[2], 10);
if (isNaN(i)) {
  console.log('Not a number');
} else {
  console.log('My number: ' + i);
}

File: 0x12-javascript-warm_up/Rusty/5-to_integer.rs
/* prints 'My number:' and the first arguemnt that can be converted into an int */
use std::env;

fn main() {
	let args: Vec<String> = env::args().collect();

	print!("My number: ");
	if args.len() > 1 {
		match args[1].parse::<usize>() {
			Ok(n) => { println!("{}", args[1]); n },
			Err(_) => { println!(""); 0 },
		};
	}
}
